README.md

```diff
@@ -187,7 +189,7 @@ The name of the application pool for the application.
 
 #### `virtual_directory`
 
-The IIS Virtual Directory to convert to an application on create.
+The IIS Virtual Directory to convert to an application on create. Similar to iis_application, iis_virtual_directory uses composite namevars.
```

Comment:
Nit: I'd change the wording from `Similar to` to `Like` or `As with` - Similar implies that the implementation is different but doesn't go on to explain how.

Reply:
Out of scope for this PR. The commit that caused this to show up in the diff has been rebased out of this PR.